util.c

```diff
@@ -2303,22 +2306,27 @@ void manage_fork_fds(int close_all)
 {
 	DIR *dir;
 	struct dirent *dirent;
+	int fd;
 
 	close(0);
-	open("/dev/null", O_RDWR);
+	fd = open("/dev/null", O_RDWR);
+	if (!is_fd_valid(fd))
+		pr_err("open /dev/null failed.\n");
+	if (fd > 2)
+		close_fd(&fd);
 
```

Comment:
If we cannot open /dev/null (theoretical case) we should not close streams. What about?

```c
/* Close all opened descriptors if needed and redirect streams to /dev/null. If we cannot open
 * /dev/null then do not change them.
 * For debug purposed, leave STDOUT and STDERR untouched
 * Returns:
 *	1- if any error occurred
 *	0- otherwise
 */
void manage_fork_fds(int close_all)
{
	DIR *dir;
	struct dirent *dirent;
	int fd = open("/dev/null", O_RDWR);

	if (is_fd_valid(fd)) {
		dup(fd, 0)
#ifndef DEBUG
		dup2(fd, 1);
		dup2(fd, 2);
		close_fd(&fd);
#endif
	}
```